src/scripts/trainers/Trainer.ts

```diff
@@ -17,4 +16,8 @@ class Trainer {
         return `assets/images/trainers/${imageName}.png`;
     }
 
+    public getTeam() {
```

Comment:
Same reasoning as gym pokemons
```suggestion
    get team(): GymPokemon[] {
```